frontend/src/pages/meditation/components/meditation-player/meditation-player.tsx

```diff
@@ -22,11 +26,14 @@ const MeditationPlayer: React.FC = () => {
   });
   const { id: meditationEntryId } = useParams<{ id: string }>();
   const [searchParameters] = useSearchParams();
+  const { setIsSidebarShow } = useSidebarState();
+  const navigate = useNavigate();
   const timerDuration = Number(
     searchParameters.get(AppQueryStringKey.TIMER_DURATION),
   );
 
   const [trackIndex, setTrackIndex] = useState(TRACK_FIRST_INDEX);
+  const isMobileDimension = useMediaQuery(MOBILE_DIMENSION);
```

Comment:
Cannot we handle it with css media queries?

Reply:
I can put something like that: `    const handleBackButtonPress = useCallback(() => {
    navigate(AppRoute.MEDITATION);

    if (window.innerWidth < 768) {
      setIsSidebarShow(false);
    }
  }, [navigate, setIsSidebarShow])`
  But for me, existing implementation is better.

Reply:
I can't do this only by scss, as I need to know whether it needs to add "?sidebarMode=show" to URL, when I click on this button. Because in case it is a list in desktop dimension I don't need this button, if it is mobile I need it, if I don't add "?sidebarMode=show" in mobile I can't see this list component.
So, yesterday I tried to deal with it with scss, but it doesn't work as expected.